<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XScreenClass</class>
 <widget class="QWidget" name="XScreenClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XScreen</string>
  </property>
  <property name="windowIcon">
   <iconset resource="XScreen.qrc">
    <normaloff>:/XScreen/logo.ico</normaloff>:/XScreen/logo.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"> #back
{
background-color:rgb(53,53,53);
border-radius:15px;
}
 #all
{
background-color:rgb(35,35,35);
}
#title
{
color:rgb(0,170,255);
font:75 11pt &quot;黑体&quot;;
}
#timeLabel
{
color:rgb(0,170,255);
font:75 20pt &quot;黑体&quot;;
}</string>
  </property>
  <widget class="QWidget" name="back" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>1000</height>
    </rect>
   </property>
   <widget class="QPushButton" name="closeBtn">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>10</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/XScreen/close.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="minBtn">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>10</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/XScreen/mini.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="all" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>801</width>
      <height>361</height>
     </rect>
    </property>
    <widget class="QPushButton" name="recordBtn">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>60</y>
       <width>96</width>
       <height>96</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:!hover
{
background-image: url(:/XScreen/record_normal.png);
}
QPushButton:hover
{
background-image: url(:/XScreen/record_hot.png);
}
QPushButton:pressed
{
background-image: url(:/XScreen/record_pressed.png);
background-color:rgba(255, 255, 255, 0);
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="urlEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>601</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>D:\video</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="widthEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>1280</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="heightEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>200</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>720</string>
     </property>
    </widget>
    <widget class="QLabel" name="timeLabel">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>160</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>000:00</string>
     </property>
    </widget>
    <widget class="QLabel" name="infoLabel">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>290</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>开发维护：夏佳权 2020</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="fpsEdit">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>200</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QLabel" name="fpsLabel">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>200</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>帧率:</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>XScreen屏幕录像机V1.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/XScreen/logo.png);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="XScreen.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeBtn</sender>
   <signal>clicked()</signal>
   <receiver>XScreenClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>775</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>781</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minBtn</sender>
   <signal>clicked()</signal>
   <receiver>XScreenClass</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>735</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>728</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>recordBtn</sender>
   <signal>clicked()</signal>
   <receiver>XScreenClass</receiver>
   <slot>Record()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>690</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>719</x>
     <y>617</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Record()</slot>
 </slots>
</ui>
